src/test/ui/hrtb/hrtb-perfect-forwarding.rs

```diff
@@ -29,22 +29,22 @@ impl<'a,X,F> Bar<X> for &'a mut F
 {
 }
 
-fn no_hrtb<'b,T>(mut t: T)
-    where T : Bar<&'b isize>
-{
-    // OK -- `T : Bar<&'b isize>`, and thus the impl above ensures that
-    // `&mut T : Bar<&'b isize>`.
-    no_hrtb(&mut t);
-}
-
-fn bar_hrtb<T>(mut t: T)
-    where T : for<'b> Bar<&'b isize>
-{
-    // OK -- `T : for<'b> Bar<&'b isize>`, and thus the impl above
-    // ensures that `&mut T : for<'b> Bar<&'b isize>`.  This is an
-    // example of a "perfect forwarding" impl.
-    bar_hrtb(&mut t);
-}
+//fn no_hrtb<'b,T>(mut t: T)
```

Comment:
@nikomatsakis Did you mean to leave this commented out?

Reply:
Um probably not, thanks!